src/components/AudioTrack/AudioTrack.vue

```diff
@@ -13,28 +17,35 @@
       {{ audio.category }}
     </div>
 
-    <div class="waveform-section flex flex-row gap-2">
+    <div class="flex flex-row gap-2">
       <PlayPause
         v-if="isCompact"
         class="flex-shrink-0"
         :is-playing="isPlaying"
         :disabled="!isReady"
         @toggle="setPlayerState"
       />
-      <Waveform
-        :class="isCompact ? 'h-20' : 'h-30'"
-        :is-ready="isReady"
-        :current-time="currentTime"
-        :duration="duration"
-        :peaks="audio.peaks"
-        :show-duration="isCompact"
-        @seeked="setPosition"
-      />
+      <div
+        class="waveform-section"
+        @keypress.enter="setPlayerState(!isPlaying)"
+        @keypress.space="setPlayerState(!isPlaying)"
```

Comment:
Why not add the `@keypress` events on `Waveform` directly? We could omit the extra div.

Reply:
Vue doesn't pass native events down to custom components without the `.native` modifier, which is advised against and removed in Vue 3.

https://vuejs.org/v2/guide/components-custom-events.html#Binding-Native-Events-to-Components

Reply:
Thanks for the docs link and sorry for the incorrect suggestion!